usaspending_api/awards/models.py

```diff
@@ -455,7 +455,7 @@ class TransactionAssistance(DataSourceTrackedModel):
     uri = models.TextField(blank=True, null=True)
     cfda_number = models.TextField(blank=True, null=True, verbose_name="CFDA Number")
     cfda_title = models.TextField(blank=True, null=True, verbose_name="CFDA Title")
-    cfda = models.ForeignKey(CFDAProgram, models.DO_NOTHING, null=True)
+    cfda = models.ForeignKey(CFDA, models.DO_NOTHING, related_name="related_assistance", null=True)
```

Comment:
I like these explicitly-named relationships! Do you think it makes sense to take this once step further and be specific about the particular assistance attribute we're referencing here (e.g., `related_cfda`)?